### PR TITLE
Update Quarkus version

### PR DESCRIPTION
**Pull request by carlesarnal**

### CODE REVIEW

**Comment by carlesarnal** [comment]
@EricWittmann this is good to go. I think we should do a new release once we get this merged. The tenant-manager-client here is required in the fleet-manager so we can also update it to Q2.

File: app/pom.xml [hunk 1]
```diff
@@ -98,6 +93,7 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-kotlin</artifactId>
+            <scope>test</scope>
```
Comment (EricWittmann) [inline app/pom.xml:96]: Good!

File: app/src/test/java/io/apicurio/registry/events/KafkaEventsTest.java [hunk 2]
```diff
@@ -57,8 +57,8 @@
  */
 @QuarkusTest
 @TestProfile(KafkaEventsProfile.class)
-@DisabledIfSystemProperty(named = "kafka.storage", matches = "true")
 @Tag(ApicurioTestTags.DOCKER)
+@Disabled
```
Comment (EricWittmann) [inline app/src/test/java/io/apicurio/registry/events/KafkaEventsTest.java:61]: I guess this needs some attention?

File: storage/kafkasql/pom.xml [hunk 3]
```diff
@@ -42,6 +42,10 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-apicurio-registry-avro</artifactId>
+        </dependency>
```
Comment (EricWittmann) [inline storage/kafkasql/pom.xml:48]: Why?

File: storage/sql/pom.xml [hunk 4]
```diff
@@ -36,6 +31,11 @@
 
         </dependency>
 
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-apicurio-registry-avro</artifactId>
+        </dependency>
```
Comment (EricWittmann) [inline storage/sql/pom.xml:37]: Also why?

**Review by EricWittmann — APPROVED** [review]
LGTM

File: app/src/test/java/io/apicurio/registry/events/KafkaEventsTest.java (hunk 2 above)
Comment (carlesarnal) [reply app/src/test/java/io/apicurio/registry/events/KafkaEventsTest.java:61]: It does but not in this PR. This is something that wasn't working before already. Same with HttpEventsTests, it's not working and it should be addressed, but I don't think it should be done here.

**Pull request merged**